docs/organizations/managing-security-and-risk.md

```diff
@@ -239,6 +236,21 @@ The following table defines finding severities and the number of days to the dea
 
 <small><sup>1</sup> Those listed are the default Jira priority names. If you rename a default Jira priority, it keeps the correct mapping.</small>
 
+### Customize deadlines {: id="item-configurable-deadlines"}
+
+!!! info "This feature is available only to [organization admins and organization managers](../organizations/roles-and-permissions-for-organizations.md)."
+
+You can configure your findings deadline clicking on the "Configure SLAs" button, on the right corner of the page.
+
+![Security and risk management SLAs configure](images/security-risk-management-slas-configure.png)
+
+By opening the configuration modal, you'll be able to input your deadline preferences for each severity. Each deadline requires at least 1 day and can't exceed the maximum of 9999 days to be solved.
+
+![Security and risk management SLAs configuration modal](images/security-risk-management-slas-modal.png)
+
+As soon as changes are saved, your findings statuses will be updated accordingly, whether they are opened or closed.
```

Comment:
I don't think this is true. Closed items have the `closed_at` date defined and that determines if items are considered closed or not. Changing the SLAs does not change this date.

So I would suggest:
```suggestion
As soon as changes are saved, your open findings statuses will be updated accordingly.
```

Reply:
For completeness sake, I'm leaving this comment here although no changes are required.

I now think that statuses for closed findings are changed as well, I misunderstood the concept of statuses. However, since closed findings are usually "hidden" behind default filters (thus less visible), and since changing status of closed issues might be considered strange, I would still only mention that open findings statuses change.